### PR TITLE
Updated ci workflows

### PR DESCRIPTION
**Pull request by KSDaemon**
added gofmt check, git commits and PR checks

### What kind of change does this PR introduce?

Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

If you're unsure about any of these, don't hesitate to ask teammates. We're here to help!

- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] Overall test coverage is not decreased.
- [ ] All new and existing tests passed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: core/helpers_test.go [hunk 1]
```diff
@@ -111,11 +111,20 @@ func TestProgressArgsKWArgs(t *testing.T) {
 		kwargs         wamp.Dict
 		expectedResult string
 	}{
-		{wamp.List{"test", 1, true, "1.0"}, wamp.Dict{}, `args: ["test",1,true,"1.0"]  
+		{
+			wamp.List{"test", 1, true, "1.0"},
+			wamp.Dict{},
+			`args: ["test",1,true,"1.0"]
```
Comment (muzzammilshahid) [inline core/helpers_test.go:117]:
```suggestion
			`args: ["test",1,true,"1.0"]  
```
This will make ci happy

**Review by bboozzoo — APPROVED** [review]
LGTM

**Comment by KSDaemon** [comment]
Also `go test -race -cover ./...` fails....

File: core/helpers_test.go (hunk 1 above)
Comment (muzzammilshahid) [reply core/helpers_test.go]: yeah that is weird. We can make it better by changing the actual function. I will do that in follow up.

Comment (KSDaemon) [reply core/helpers_test.go]: 👍🏻

**Pull request merged**